docs/features/blocks/contentblocks/latestpages.rst

```diff
@@ -0,0 +1,29 @@
+Latest Pages Block
+==================
+
+Creates a list of the most recently published pages with a specified length. 
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **Parent Page** - Shows a preview of pages that are children of the selected page. Uses ordering specified in the page’s LAYOUT tab.
+
+* **Classified By** - Filters which pages are displayed by the classifier that you selected
+
+* **Show Body Preview** - If selected, shows a preview of what the page contains 
+
+* **Number of Pages to Show** - Limits how many pages are displayed to the number that you selected
```

Comment:
Can you also add a note here, that there are a few built-in templates for this block under Advanced settings?

Reply:
note added